driver/pelican/demandResponse.go

```diff
@@ -0,0 +1,142 @@
+package main
+
+import (
+	"encoding/xml"
+	"fmt"
+	"time"
+)
+
+type DR_EVENT_STATUS int
+
+const (
+	DR_EVENT_UNKNOWN        DR_EVENT_STATUS = -1
+	DR_EVENT_NOT_CONFIGURED DR_EVENT_STATUS = iota
+	DR_EVENT_UNUSABLE
+	DR_EVENT_INACTIVE
+	DR_EVENT_ACTIVE
+)
+
+type DR_EVENT_TYPE int
+
+const (
+	DR_EVENT_UNRECOGNIZED DR_EVENT_TYPE = -1
+	DR_EVENT_NORMAL       DR_EVENT_TYPE = iota
+	DR_EVENT_MODERATE
+	DR_EVENT_HIGH
+	DR_EVENT_SPECIAL
+)
+
+type ADREventWrapperAPI struct {
+	Success int         `xml:"success"`
+	Message string      `xml:"message"`
+	Event   ADREventAPI `xml:"attribute"`
+}
+
+type ADREventAPI struct {
+	End    string `xml:"OpenADREventEnd"`
+	Start  string `xml:"OpenADREventStart"`
+	Status string `xml:"OpenADRStatus"`
+	Type   string `xml:"OpenADREventType"`
+}
+
+type ADREvent struct {
```

Comment:
Can we annotate this struct so that the msgpack serialization is compliant with the [XBOS DR Interface](https://github.com/SoftwareDefinedBuildings/XBOSDocs/blob/master/driver_xbos_demand_response.md)?

Should be able to do this like

```go
type ADREvent struct {
    EventEnd int64 `msgpack:"event_end"`
    // etc
```